M2/Macaulay2/e/aring-zzp-ffpack.hpp

```diff
@@ -17,8 +17,9 @@
 
 // this fixes a weird problem in the package "openblas" of Arch Linux, which somehow fails to declare this function:
 extern "C" void openblas_set_num_threads(int num_threads);
```

Comment:
Not relevant to your changes, but seeing this reminded me that we should probably also call [`flint_set_num_threads`](http://flintlib.org/doc/threading.html) to use threading there as well.